odoorpc/tests/test_login.py

```diff
@@ -55,3 +58,15 @@ def test_logout_without_login(self):
         )
         success = odoo.logout()
         self.assertFalse(success)
+
+    def test_logout_closing(self):
+        odoo = odoorpc.ODOO(
+            self.env['host'],
+            protocol=self.env['protocol'],
+            port=self.env['port'],
+            version=self.env['version'],
+        )
+        odoo.login(self.env['db'], self.env['user'], self.env['pwd'])
+        with closing(odoo):
+            self.assertTrue(odoo._check_logged_user())
```

Comment:
`_check_logged_user()` returns nothing but raises an exception if the user is not logged.
```suggestion
            odoo._check_logged_user()
```